packages/website/src/components/ESQueryFilter.module.css

```diff
@@ -0,0 +1,27 @@
+.searchContainer {
+  display: flex;
+  margin: 0 0 0.5rem 0;
+  position: sticky;
+  top: 0;
+  left: 0;
+}
+
+.search {
+  -webkit-appearance: none;
+  -moz-appearance: none;
```

Comment:
We're good to no longer use these: https://caniuse.com/css-appearance

```suggestion
```

A good followup might be to add [Stylelint](https://stylelint.io) to our builds, since we've got a good amount of CSS now.